src/sections/Navigation/utility/ScrollspyMenu.js

```diff
@@ -51,12 +65,12 @@ const ScrollspyMenu = ({ menuItems, ...props }) => {
                             </div>
                             <div className="nav-display">
                                 <div className="nav-div">
-                                    <a href={active.div1.path}><img src={active.div1.src} className="nav-img" /></a>
-                                    <div> {active.div1.descr} </div>
+                                    <Link href={blogData[active.name].nodes[0].fields.slug}><img src={blogData[active.name].nodes[0].frontmatter.thumbnail.publicURL} className="nav-img" /></Link>
```

Comment:
After using childImageSharp. `Image` component can be used

Reply:
What are the available image sizes @Tanuj22

Reply:
Any size will work